mesheryctl/cmd/root.go

```diff
@@ -72,9 +72,7 @@ var rootCmd = &cobra.Command{
 		if b {
 			versionCmd.Run(nil, nil)
 		}
-		if len(args) == 0 {
-			log.Print(cmdDetails)
-		}
```

Comment:
@ashiscs any specific reason to remove these line of codes?

Reply:
As beacuse while printing we only want the mesheryctl version as a output, but this lines of code were also printing about meshery help command which is not necessary though!!

Reply:
But, we do need to add this as because, if a user fails to give argument after mesheryctl then it will print all the details related to meshery help..

Reply:
Okay will you try to solve that issue with this PR?

Reply:
Sure..